lightning/cookie.py

```diff
@@ -0,0 +1,59 @@
+from __future__ import annotations
+
+from datetime import datetime
+from lightning_adapter.models import Cookie as RawCookie
+
+
+class Cookie(dict):
+
+    def __init__(self, *, name: str, value: str, path: str = None,
+                 domain: str = None, secure_only: bool = False,
+                 http_only: bool = False, expires: datetime.timestamp = None,
+                 same_site_policy: str = None):
+
+        self._raw = RawCookie(name=name, value=value, path=path,
+                              domain=domain, secure=secure_only, http_only=http_only,
+                              expiry=expires, same_site=same_site_policy
+                              )
+
+    def raw(self):
+        return self._raw
+
+    @property
+    def name(self):
+        return self._raw.name
+
+    @property
+    def value(self):
+        return self._raw.value
+
+    @property
+    def path(self):
+        return self._raw.path
+
+    @property
+    def domain(self):
+        return self._raw.domain
+
+    @property
+    def secure_only(self):
+        return self._raw.secure
+
+    @property
+    def http_only(self):
+        return self._raw.http_only
+
+    @property
+    def expires(self):
+        return self._raw.expiry
+
+    @property
+    def same_site_policy(self):
```

Comment:
Probably there is a generated SameSitePolicy enum. If so, return it here.

Reply:
In docs it is described as str